editor/components/block-list/block.js

```diff
@@ -385,6 +394,15 @@ export class BlockListBlock extends Component {
 		}
 	}
 
+	onSelectionChange() {
+		const selection = window.getSelection();
+		const range = selection.getRangeAt( 0 );
```

Comment:
Seeing an error sometimes:

```
block.js:399 Uncaught DOMException: Failed to execute 'getRangeAt' on 'Selection': 0 is not a valid index.
    at BlockListBlock.onSelectionChange (http://localhost:8000/wp/wp-content/plugins/gutenberg-dev/editor/build/index.js?ver=1517913835:23499:26)
onSelectionChange @ block.js:399
```

Also, since we're in a TinyMCE component, we could use `editor.selection`?

Reply:
We're not in TinyMCE :)

Do you have a way to consistently reproduce the error? which browser?

Reply:
> We're not in TinyMCE :)

Mixed up components. :)

Reply:
Yes, select a paragraph, then click on the border outside the `RichText` region.

Reply:
(Chrome)

Reply:
Thanks this should be fixed now :)

Reply:
Oh I just saw that `rangeCount` is not supported widely yet. Looking for an alternative here.